neptune-core/src/application/rpc/server.rs

```diff
@@ -4163,9 +4179,96 @@ impl RPC for NeptuneRPCServer {
             .map(|tx| &tx.kernel)
             .cloned())
     }
+
+    // Crashes if `tx_ix` out of its bound. https://github.com/Neptune-Crypto/neptune-core/issues/816
```

Comment:
This crash condition should live on the docstring above, where the function is declared and the trait defined.